### DISCUSSION TITLE
Klipper update to the latest commit

### DISCUSSION THREAD

**Discussion by Newcal** [Desuuuu/klipper#126] [Ideas]
Gday Desuuuu,

Is there any chance to update DGUS reloaded to the latest commit to enjoy the last feature ?

I'd like to use exclude object but this one is obviously not supported.

Is it possible to make it happen ?

Thanks a lot for the good work anyway.
JS

<img width="401" alt="Screen Shot 2022-07-20 at 07 34 54" src="https://user-images.githubusercontent.com/18645222/179865809-6c4b9912-3fbc-406c-a540-ffb0297ad81c.png">

**Comment by Desuuuu** [comment]
The dgus-reloaded branch is now up-to-date.

**Reply by Newcal** [reply]
Legend ! Thanks mate ! Do we need to flash the screen again with it ?

Cheers

**Reply by Desuuuu** [reply]
This isn't needed, as there isn't any new feature.

**Reply by Thinkersbluff** [reply]
Do you still update this fork from Klipper3D/master?

I forked your fork to support a CR6-specific display interface, but I wonder whether in doing so, I am also now locked-in to Klipper 0.10.0 (as last modified, Sept 2022)

I tried merging 0.11.0 directly into my fork but GitHub says conflicts must first be resolved, and that conflict resolution is clearly presently beyond my limited skill set.
